### DISCUSSION TITLE
Add a batch file which starts FHGearbox together with FH5

### DISCUSSION THREAD

**Discussion by FlazeIGuess** [GinoLin980/Forza-Horizon-realistic-gearbox#6] [Ideas]
### I've found a simple way to start FHGearbox and FH5 through Steam!

First go to your FH5 Directory, press right click and create a new text file.

Paste this "code" in:
@echo off
cd /d "%~dp0"
start "" "ForzaHorizon5.exe"

for %%f in ("FHGearbox*.exe") do (
    start "" "%%f"
    goto :end
)

:end
exit

after that save the .txt file and rename it to "(textfilename).bat" instead of "(textfilename).txt" 

now copy the FHGearbox.exe in the FH5 Directory.

then right click Forza Horizon 5 in steam and click on properties under start options paste the path to your batch file:

for example:
"D:\SteamLibrary\steamapps\common\ForzaHorizon5\StartForza.bat" %command%

now leave one space and add "%command%"

Now everytime you start FH5, FHGearbox starts with it! :)

**Comment by GinoLin980** [comment]
@FlazeIGuess Thanks for the useful suggestion. 
~You can open up an issue labeled "Enhancement" if you would like to be the contributor shown on the main page of this repo.~(I found out issues can't be included as contributor :( 
___
## Here is the more detailed introduction and improved command.

### Reminder, it is a great habit to understand the command before running it.

First, copy the command below and paste to any text editor

```batch
@echo off

cd /d "%~dp0"
start "" "{ABSOLUTE}\{PATH}\{TO}\ForzaHorizon5.exe"

for %%f in ("{ABSOLUTE}\{PATH}\{TO}\FHGearbox*.exe") do (
	start "" "%%f"
	goto :end
)

:end
exit
```

Then, find your Forza Horizon path and copy it, and replace the `{ABSOLUTE}\{PATH}\{TO}\ForzaHorizon5.exe` with it(double quotes should be remained)

Do the same thing to the `{ABSOLUTE}\{PATH}\{TO}\FHGearbox*.exe`

Second, do one of the following

### Method 1
Create a `.txt` at anywhere you want, copy the modified command, then paste the command, then click `File > Save as...` and change the extension to `.bat`

### Method 2
Open up a CMD at your favored directory, then copy the following command to CMD and **don't run it**

```batch
powershell -command "Get-Clipboard" > {YOUR_FILENAME}.bat
```

then, copy the modified command and hit enter on your CMD.

#### DONE!!! Just click on the batch file created and it'll open the game and the Gearbox App together.

**Reply by FlazeIGuess** [reply]
You coule add to the introduction, that If you have Forza Horizon 5 on Steam that it's also possible to run FHGearbox together with FH5 through Steam :)
